### ISSUE TITLE
Building on rpi1 / armv6

### ISSUE THREAD

**Issue by joolswills** [loganmc10/GLupeN64#98]
Should glupen64plus build on armv6 / rpi ?

I thought it did before, but perhaps I was mistaken.

Currently it will fail due to this in the makefile (It will add `-mstackrealign -DARCH_MIN_SSE2 -msse -msse2` for armv6 as no NEON)

(building with make platform=rpi1 - and setting my cflags accordingly.

```
ifeq ($(HAVE_NEON), 1)
   COREFLAGS += -DHAVE_NEON -D__ARM_NEON__ -D__NEON_OPT
else
   COREFLAGS += -mstackrealign -DARCH_MIN_SSE2 -msse -msse2
endif
```

not to worry if not, but I will disable for armv6 in retropie then.

**Comment by joolswills** [comment]
does build on the rpi1 at least. I will do a quick hack in the retropie-script for now.

**Comment by loganmc10** [comment]
No it should work, I have really lazy logic in he Makefile right now that assumes neon=arm, I'll fix it up

**Comment by loganmc10** [comment]
Ok I don't have an rpi1 to test it on, but this should be fixed by 34d2c0a13b75839ec0536ec15cb8adeec9cea9e9

**Comment by joolswills** [comment]
All good - thanks :)
